packages/applications-service-api/prisma/schema.prisma

```diff
@@ -74,6 +74,24 @@ model Representation {
   @@index([caseReference])
 }
 
+model ServiceUser {
+  id               Int      @unique @default(autoincrement())
+  serviceUserId    Int      @id
+  caseReference    String?
+  firstName        String?
+  middleName       String?
+  lastName         String?
+  email            String?
+  webAddress       String?
+  phoneNumber      String?
+  organisationName String?
+  organisationType String?
+  role             String?
```

Comment:
This is `jobTitle` in BO db. Same question as above regarding consistency across services